### PR TITLE
Allows running on a .NETFramework based TFM to ease the dotnet CLI transition

### PR DESCRIPTION
**Pull request by davidfowl**
- This will allow us to run on both DNX and the .NET CLI and be closer
to what our shipping product looks like as we transition away from DNX
projects.

/cc @anurse @troydai @pakrym

### CODE REVIEW

File: src/Microsoft.Dnx.Host/FrameworkNameUtility.cs [hunk 1]
```diff
@@ -35,6 +35,31 @@ internal static bool TryParseFrameworkName(string frameworkName, out FrameworkNa
                 parsed = new FrameworkName(FrameworkNames.LongNames.Dnx, new Version(4, 6));
                 return true;
             }
+            else if (frameworkName == FrameworkNames.ShortNames.Dnx461)
```
Comment (pakrym) [inline src/Microsoft.Dnx.Host/FrameworkNameUtility.cs]: Dictionary?

Comment (davidfowl) [inline src/Microsoft.Dnx.Host/FrameworkNameUtility.cs]: It's not like this code needs to be maintained :smile:

Comment (pakrym) [inline src/Microsoft.Dnx.Host/FrameworkNameUtility.cs]: Then why review? :)

Comment (davidfowl) [inline src/Microsoft.Dnx.Host/FrameworkNameUtility.cs]: Extra set of eyes, plus following process :+1:

**Comment by pakrym** [comment]
What about dotnet*?

**Comment by davidfowl** [comment]
@pakrym You never run **dotnet** but there is a new TFM that will replace dnxcore50, the current working title is **netstandardapp**

**Comment by pakrym** [comment]
:shipit:

**Pull request merged**